src/Pkg.jl

```diff
@@ -5,8 +5,14 @@ import REPL
 using REPL.TerminalMenus
 
 depots() = Base.DEPOT_PATH
-logdir() = joinpath(depots()[1], "logs")
-devdir() = get(ENV, "JULIA_PKG_DEVDIR", joinpath(depots()[1], "dev"))
+function depots1()
```

Comment:
This should really be renamed to `user_depot` since that's a much more meaningful name for it.